src/main/java/ru/practicum/shareit/exception/ErrorHandler.java

```diff
@@ -0,0 +1,33 @@
+package ru.practicum.shareit.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ErrorHandler {
+    @ExceptionHandler
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public ErrorResponse handleNotFoundException(final NotFoundException e) {
```

Comment:
Лучше не использовать однобуквенные названия переменных, они делают код менее читаемым

```suggestion
    public ErrorResponse handleNotFoundException(final NotFoundException exception) {
```